### PR TITLE
openshift/prometheus-operator: use base images from repo

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-operator-master-images

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-operator-master-images-test-unit

**Comment by danielmellado** [comment]
Just checked https://github.com/openshift/release/pull/45914, thanks for creating this one on the same lines! /lgtm

File: ci-operator/config/openshift/prometheus-operator/openshift-prometheus-operator-master.yaml [hunk 1]
```diff
@@ -1,36 +1,15 @@
...
-  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.20-openshift-4.15
 build_root:
   from_repository: true
 canonical_go_repository: github.com/coreos/prometheus-operator
 images:
 - dockerfile_path: Dockerfile.ocp
   from: base
```
Comment (danielmellado) [inline ci-operator/config/openshift/prometheus-operator/openshift-prometheus-operator-master.yaml:6]: shouldn't we remove this line, as in https://github.com/openshift/release/pull/45914/files example? and overall all the from: base as there's no more base defines?

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-operator-master-images

File: ci-operator/config/openshift/prometheus-operator/openshift-prometheus-operator-master.yaml (hunk 1 above)
Comment (simonpasquier) [reply ci-operator/config/openshift/prometheus-operator/openshift-prometheus-operator-master.yaml]: yep I realize this after running the rehearsal.

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-operator-master-images

**Comment by simonpasquier** [comment]
/pj-rehearse pull-ci-openshift-prometheus-operator-master-test-unit

**Review by simonpasquier — COMMENTED** [review]
@danielmellado can you have another look please?

**Comment by machine424** [comment]
Thanks @simonpasquier. LGTM, will just run e2e-agnostic-cmo as unit and images are already passing in https://github.com/openshift/prometheus-operator/pull/258

/pj-rehearse pull-ci-openshift-prometheus-operator-master-e2e-agnostic-cmo

**Comment by machine424** [comment]
/tgtm
(do not pay attention to the failing `ci/rehearse/openshift/prometheus-operator/master/e2e-agnostic-cmo` )

**Comment by rexagod** [comment]
> well I don't think the test will pass in master, I don't know if we can run the e2e-agnostic test from https://github.com/openshift/prometheus-operator/pull/258 via a rehearsal...

@machine424 Did you cancel the `e2e-agnostic-cmo` job due to the fact that we may not be able to actually run that from here (as you stated above)? Not sure if I get why this was cancelled.

**Comment by machine424** [comment]
Yep, I canceled it with the `/pj-rehearse abort`

(I was willing to run pull-ci-openshift-prometheus-operator-`openshift-bot:art-consistency-openshift-4.15-prometheus-operator`-e2e-agnostic-cmo but I don't think it's possible (even though I could be a goof feature to have))

**Pull request merged**